### DISCUSSION TITLE
Interrupting compilation by navigating through project files

### DISCUSSION THREAD

**Discussion by jcasale** [texstudio-org/texstudio#3894] [Q&A] [answered]
I have a large document with 400 pages and several longtblr and tikz entries. The document is split across several `input` and `subfiles` imports. My builds take between 2 and 3 runs to complete, and up to 15 minutes each on a fast computer.

I typically do other work while this long process runs, but sometimes I browse through the project and read it to perform content and grammar checks.

If I navigate off the root document while compiling, the compilation never repeats the necessary times to complete the build. When the current build iteration completes, the PDF is opened regardless of whether the need to recompile is present.

How can I prevent this? I'd like to see a build invocation remember it's starting arguments and maintain its requirements until complete.

Thanks!

**Comment by sunderme** [comment, marked as answer]
You can put a feature request, but I doubt that it can be easily solved.
The better alternative is to use `latexmk`as build tool.
